src/main/java/frc/robot/commands/ArmReverse.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Arm;
+
+public class ArmReverse extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final Arm m_subsystem;
```

Comment:
this variable should probably be named 'm_arm' or 'm_armSubsystem'.
its doesn't tell us anything if its just named 'subsystem'.